_posts/2024-01-31-good-bad-web-components.md

```diff
@@ -254,7 +260,7 @@ In some respects it feels like the web platform chased the early clientside rend
 	* [{% indieAvatar "https://lit.dev/" %}Lit](https://lit.dev/)
 	* [{% indieAvatar "https://enhance.dev/" %}Enhance](https://enhance.dev/)
 	* [{% indieAvatar "https://www.11ty.dev/" %}WebC](https://www.11ty.dev/docs/languages/webc/)
-* React has the worst [`custom-elements-everywhere.com`](https://custom-elements-everywhere.com/) compatibility test score. If you’re deep in React-world, look at using [{% indieAvatar "https://preactjs.com/" %}Preact](https://preactjs.com/) to provide a more future-friendly and compatible experience.
+* React (versions before v19) has the worst [`custom-elements-everywhere.com`](https://custom-elements-everywhere.com/) compatibility test score. If you’re deep in React-world, look at using React v19 or [{% indieAvatar "https://preactjs.com/" %}Preact](https://preactjs.com/) to provide a more future-friendly and compatible experience.
```

Comment:
if [this PR will wait on React 19 release](https://github.com/zachleat/zachleat.com/pull/53#issuecomment-2339251438), then this would be the more minimal diff

```suggestion
* Old React versions have the worst [`custom-elements-everywhere.com`](https://custom-elements-everywhere.com/) compatibility test score. If you’re deep in React-world, upgrade to the latest React version or [{% indieAvatar "https://preactjs.com/" %}Preact](https://preactjs.com/) to provide a more future-friendly and compatible experience.
```